ui/src/components/textInputDialog.tsx

```diff
@@ -8,6 +8,7 @@ import { ChangeEvent, ReactNode, useState } from "react";
 
 type textInputDialogProps = {
```

Comment:
Sorry, I missed this on the previous PR. The first letter of `textInputDialogProps` should be capitalized since it's a type, just like components (so `TextInputDialogProps`). Functions and variables start with a lower case letter. Please make another quick PR to rename it.

Reply:
Done in gy/dialogName this branch.